.github/workflows/master.yml

```diff
@@ -107,7 +107,7 @@ jobs:
             pyarrow-version: 1.0.1
           - python-version: 3.8
             spark-version: 3.0.1
-            pandas-version: 1.1.5
+            pandas-version: 1.2.0
```

Comment:
Shall we maybe use `1.2.1` btw?

Reply:
@itholic up to you, we can do it separately.

Reply:
Sure, I'd upgrade this here.